torch/_inductor/codecache.py

```diff
@@ -1304,6 +1304,13 @@ def load(  # type: ignore[no-untyped-def]
                 counters["inductor"]["fxgraph_cache_hit"] += 1
                 cache_state = "hit"
                 cache_event_time = time_ns()
+                if (
+                    dist.distributed_c10d.is_initialized()
+                    and (time_taken_ns := compiled_graph._time_taken_ns) is not None
```

Comment:
Am I reading this right, that this will just be enabled if we land this PR? (Should we put a env var or killswitch on it? I asume we'll want to smoke test it witn an internal job?).

Reply:
Caching is not being used with distributed right now, so this code will not be used anywhere, which is why I dont think we need a killswitch